src/features/listing-builder/components/Modals/Survey.tsx

```
@@ -0,0 +1,194 @@
+
+interface Props {
+  open: boolean;
+  setOpen: (open: boolean) => void;
+}
+
+const surveyId =
+  process.env.NEXT_PUBLIC_VERCEL_ENV === 'production'
+    ? '0197eaf7-673f-0000-7880-4194601272e7'
+    : '0197ea95-a533-0000-dc94-0ee7420bf670';
+
+export function Survey({ open, setOpen }: Props) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Survey'. Consider to delete it or rename it.

'Survey' is defined here:


(lint/suspicious/noRedeclare)